.puprc

```diff
@@ -66,17 +43,14 @@
 			},
 			{
 				"file": "package.json",
-				"regex": "(\"version\": \")([^\"]+)"
+				"regex": "(\"version\": ?\")([^\"]+)"
 			},
 			{
 				"file": "readme.txt",
 				"regex": "(Stable tag: )(.+)"
 			}
 		],
-		"views": [
-			"src/views",
-			"common/src/views"
-		]
+		"views": [ "src/views" ]
```

Comment:
I'd undo this change - the goal is for version checking to be done with templates during packaging, so this will need to remain (though, I haven't ported that `tut` command over yet).